src/screen.py

```diff
@@ -21,13 +21,30 @@
 
 
 def handle_sigterm(signum, frame):
+    """
```

Comment:
Use Google Style for commenting functions. More information [here](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods).